src/index.js

```diff
@@ -246,7 +245,6 @@ class Node extends EventEmitter {
 
       this._switch.dial(peerInfo, protocol, (err, conn) => {
```

Comment:
We can just pass the callback now. Lines below have to be removed

```suggestion
      this._switch.dial(peerInfo, protocol, callback)
```

Reply:
Ah yeah, good call! I've updated those.